### PR TITLE
Community page refactor and new learning section

### PR DESCRIPTION
**Pull request by charlieegan3**
This PR refactors the community page to use content from the yaml front matter for the page. The should make it easier to update in future.

I have also added a new learning section and reordered the page a little.

### CODE REVIEW

File: docs/website/static/img/community-logos/styra-academy.png
Comment (srenatus) [inline docs/website/static/img/community-logos/styra-academy.png:1]: image

This looks a _little_ off to me... If we've got an alternative logo, that would be cool -- but it's no big deal if not.

**Review by srenatus — DISMISSED** [review]

**Review by anderseknert — APPROVED** [review]
Nice!

File: docs/website/static/img/community-logos/styra-academy.png
Comment (charlieegan3) [reply docs/website/static/img/community-logos/styra-academy.png:1]: Maybe this image? 
https://github.com/StyraInc/awesome-opa/blob/main/assets/awesome-opa.png

Comment (srenatus) [reply docs/website/static/img/community-logos/styra-academy.png:1]: but yeah, the awesome-opa logo could also look nice here

Comment (charlieegan3) [reply docs/website/static/img/community-logos/styra-academy.png:1]: Screenshot 2023-06-15 at 14 35 42

It looks like this now.

Comment (anderseknert) [reply docs/website/static/img/community-logos/styra-academy.png:1]: Sorry to come in late here, but that logo is almost certainly a trademark violation 😅 I don't think we can use it on openpolicyagent.org

**Pull request merged**